test/extension-tests/successful-build/test/toolchain.test.ts

```diff
@@ -17,7 +17,9 @@ suite('[Toolchain Substitution]', async () => {
     testEnv = new DefaultEnvironment('test/extension-tests/successful-build/project-folder', 'build', 'output.txt');
     cmt = await CMakeTools.create(testEnv.vsContext, testEnv.wsContext);
 
-    const kits = await kitsAvailableInWorkspaceDirectory(testEnv.projectFolder.location);
+    const user_kits = await readKitsFile(USER_KITS_FILEPATH);
```

Comment:
Why does this read the User kits now? I don't think it's supposed to.

Reply:
The original function (kitsAvailableInWorkspaceDirectory) reads user kits as well.

Reply:
you're right. sorry I missed that.